internal/vulnstore/postgres/gethash_integration_test.go

```diff
@@ -1,15 +1,18 @@
-//+build integration
-
 package postgres
 
 import (
+	"context"
 	"testing"
 
+	"github.com/quay/claircore/test/integration"
+
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/assert"
 )
 
 func Test_GetHash_KeyNotExists(t *testing.T) {
+	integration.Skip(t)
+	ctx := context.Background()
```

Comment:
Now that context exists, may not be a bad options to time these out so the test do not run indefinitely if there's a lock.